### PR TITLE
Explain browserify/Buffer behavior

### PR DESCRIPTION
**Pull request by paulkernfeld**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ either `le` (little-endian) or `be` (big-endian).
   pad to length, throwing if already exceeding
 * `a.toArrayLike(type, endian, length)` - convert to an instance of `type`,
   which must behave like an `Array`
-* `a.toBuffer(endian, length)` - convert to Node.js Buffer (if available)
+* `a.toBuffer(endian, length)` - convert to Node.js Buffer (if available). For compatibility with browserify and similar tools, use this instead: `a.toArrayLike(Buffer, endian, length)`
```
Comment (indutny) [inline README.md]: 80 column limit, please.

**Comment by paulkernfeld** [comment]
Okay, I updated this commit to fix the line length.

**Comment by indutny** [comment]
LGTM

**Comment by indutny** [comment]
Thank you!

**Pull request merged**